undefined reference to `pcl::segmentation::grabcut::BoykovKolmogorov::TERMINAL'

Build fails as of 1d61ddf5850b27580ecc6980b4b8c2b04b9586a7:
```
../lib/libpcl_segmentation.so.1.8.0: undefined reference to `pcl::segmentation::grabcut::BoykovKolmogorov::TERMINAL'
collect2: error: ld returned 1 exit status
make[2]: *** [bin/pcl_cluster_extraction] Error 1
make[1]: *** [tools/CMakeFiles/pcl_cluster_extraction.dir/all] Error 2
make: *** [all] Error 2
```

NOTE: only happens when setting CXXFLAGS="-std=c++11" in the cmake environment.
The offending build line:
```
cd /usr/src/pointcloud/pcl/pcl/b/tools && /usr/bin/cmake -E cmake_link_script CMakeFiles/pcl_cluster_extraction.dir/link.txt --verbose=1
/usr/bin/c++   -std=c++11  -pthread -fopenmp -g    -Wl,--as-needed CMakeFiles/pcl_cluster_extraction.dir/cluster_extraction.cpp.o  -o ../bin/pcl_cluster_extraction -rdynamic -lboost_system -lboost_filesystem -lboost_thread -lboost_date_time -lboost_iostreams -lboost_chrono -lpthread -lpthread -lm ../lib/libpcl_common.so.1.8.0 ../lib/libpcl_io.so.1.8.0 ../lib/libpcl_segmentation.so.1.8.0 ../lib/libpcl_filters.so.1.8.0 ../lib/libpcl_kdtree.so.1.8.0 ../lib/libpcl_io_ply.so.1.8.0 -lpng -lusb-1.0 ../lib/libpcl_features.so.1.8.0 ../lib/libpcl_filters.so.1.8.0 ../lib/libpcl_sample_consensus.so.1.8.0 ../lib/libpcl_search.so.1.8.0 ../lib/libpcl_kdtree.so.1.8.0 ../lib/libpcl_common.so.1.8.0 -lflann_cpp ../lib/libpcl_octree.so.1.8.0 -lboost_system -lboost_filesystem -lboost_thread -lboost_date_time -lboost_iostreams -lboost_chrono -lpthread -lm -Wl,-rpath,/usr/src/pointcloud/pcl/pcl/b/lib: 
../lib/libpcl_segmentation.so.1.8.0: undefined reference to `pcl::segmentation::grabcut::BoykovKolmogorov::TERMINAL'
```

--- a/segmentation/include/pcl/segmentation/grabcut_segmentation.h
+++ b/segmentation/include/pcl/segmentation/grabcut_segmentation.h
@@ -161,7 +161,7 @@ namespace pcl
 
         private:
           /// parents_ flag for terminal state
-          static const int TERMINAL = -1;
+          static const int TERMINAL; // -1
           /// search tree (also uses cut_)
           std::vector<std::pair<int, edge_pair> > parents_;
           /// doubly-linked list (prev, next)

--- a/segmentation/src/grabcut_segmentation.cpp
+++ b/segmentation/src/grabcut_segmentation.cpp
@@ -45,6 +45,8 @@
 #include <map>
 #include <algorithm>
 
+const int pcl::segmentation::grabcut::BoykovKolmogorov::TERMINAL = -1;
+
 pcl::segmentation::grabcut::BoykovKolmogorov::BoykovKolmogorov (std::size_t max_nodes)
   : flow_value_(0.0)
 {

